docs/sdk/server-side-sdks/java-cloud/java-cloud.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Java SDK for Cloud Bucketing
+---
+
+Welcome to the DevCycle Java SDK, which interfaces with the [DevCycle Bucketing API](/bucketing-api/#tag/devcycle).
+All requests, including user data are sent to DevCycle servers to ensure the User is bucketed correctly and will receive the correct variation.
+
+The SDK is available as a package on MavenCentral. It is also open source and can be viewed on Github.
+
+[![Maven](https://badgen.net/maven/v/maven-central/com.devcycle/java-server-sdk)](https://search.maven.org/artifact/com.devcycle/java-server-sdk)
```

Comment:
New requirement as per the slack conversations - we now want these badges at the top of _all_ the subpages for this section.

https://taplytics.slack.com/archives/C032ZGWQPCJ/p1681485912733769